README.md

```diff
@@ -67,10 +69,14 @@ For running the feature set `winery`, the setup needs additional configuration:
 | QProv-Collector-IBM         | <http://localhost:5021>               | [Link](https://github.com/UST-QuAntiL/qprov)               | [Link](https://hub.docker.com/r/planqk/qprov-collector)                                       |
 | Latex-Renderer              | <http://localhost:5030>               | [Link](https://github.com/UST-QuAntiL/latex-renderer)      | [Link](https://hub.docker.com/repository/docker/planqk/latex-renderer)                        |
 | Redis                       | <http://localhost:5040>               | [Link](https://github.com/redis/redis)                     | [Link](https://hub.docker.com/_/redis)                                                        |
+| Redis (QHana)               | <http://localhost:6379>               | [Link](https://github.com/redis/redis)                     | [Link](https://hub.docker.com/_/redis)                                                        |
 | Postgres-Multi DB           | <tcp://localhost:5060>                | [Link](https://github.com/lmm-git/docker-postgres-multi)   | [Link](https://hub.docker.com/r/lmmdock/postgres-multi)                                       |
 | Config-Server               | <http://localhost:2379>               | [Link](https://github.com/etcd-io/etcd)                    | [Link](https://quay.io/repository/coreos/etcd)                                                |
 | Winery                      | <http://localhost:8080>               | [Link](https://github.com/eclipse/winery)                  | [Link](https://hub.docker.com/r/opentosca/winery)                                             |
-| QHAna Plugin Runner         |<http://localhost:5005>                | [Link](https://github.com/UST-QuAntiL/qhana-plugin-runner) | [Link](https://github.com/UST-QuAntiL/qhana-plugin-runner/pkgs/container/qhana-plugin-runner) |
+| QHana-UI                    | <http://localhost:5003>               | [Link](https://github.com/UST-QuAntiL/qhana-ui)       | [Link](https://github.com/UST-QuAntiL/qhana-plugin-runner/pkgs/container/qhana-ui)|
+| QHana-Backend               | <http://localhost:5004>               | [Link](https://github.com/UST-QuAntiL/qhana-backend)       | [Link](https://github.com/UST-QuAntiL/qhana-backend/pkgs/container/qhana-backend)|
+| QHAna Plugin Runner (Prio)  |<http://localhost:5005>                | [Link](https://github.com/UST-QuAntiL/qhana-plugin-runner) | [Link](https://github.com/UST-QuAntiL/qhana-plugin-runner/pkgs/container/qhana-plugin-runner) |
```

Comment:
There is no longer a separate plugin runner/worker. Please adapt readme to reflect changes.